AmplifyPlugins/API/AWSAPICategoryPlugin/Reachability/NetworkReachabilityNotifier.swift

```diff
@@ -40,7 +52,6 @@ class NetworkReachabilityNotifier {
 
     deinit {
         reachability?.stopNotifier()
-        NotificationCenter.default.removeObserver(self)
```

Comment:
Xcode says it will be automatically removed, so that no need to call `removeObserver()`